angularSTEP/src/app/upload-recipe/upload-recipe.component.ts

```diff
@@ -160,4 +196,30 @@ export class UploadRecipeComponent{
       });
     this.router.navigate(['/confirm-upload']);
   }
+
+  //image functions
+
+  onFileSubmit() {
+    this.fAuth.currentUser.then(user => {
+      if (user) {
+        //this.addImage(user.uid);
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes I did, that was commented out because that is where I will implement the actual adding of images in the next PR. It was just supposed to show that that will be called when I link up the backend with the frontend